packages/firestore/src/platform_browser/browser_platform.ts

```diff
@@ -72,4 +74,23 @@ export class BrowserPlatform implements Platform {
   btoa(raw: string): string {
     return btoa(raw);
   }
+
+  randomBytes(nBytes: number): Uint8Array {
+    debugAssert(nBytes >= 0, 'Number of random bytes must not be negative');
```

Comment:
Consider including the invalid value in the failure message; this could assist in debugging should the assert ever get triggered and all we have to use for debugging is the stack track and error message.

e.g.
```js
debugAssert(nBytes >= 0, `invalid nBytes: ${nBytes}`);
```